ARMTemplates/AzFunctionApp-Consumption.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Can we add user configurable parameters here for setting up environment variables on the function? Right now, the app expects an environment variable for `PAT`, `Organization`, and `Project`.

Reply:
Yes. Will add those parameters as well.